silkworm/node/snapshots/path.hpp

```diff
@@ -90,7 +92,7 @@ class SnapshotPath {
     }
 
     [[nodiscard]] bool seedable() const {
-        return block_to_ - block_from_ == segment_size_;
+        return std::ranges::find(kDefaultSegmentSizes, segment_size()) != kDefaultSegmentSizes.cend();
```

Comment:
better use `std::ranges::contains`

Reply:
It was my first thought as well 😄, but it will be available only [since C++23](https://en.cppreference.com/w/cpp/algorithm/ranges/contains)